index.html

```diff
@@ -88,61 +88,128 @@ <h1 class="h1 text-center">About Me</h1>
-                        <div class="d-flex align-items-end flex-column">
-                            <p class="quote-name mb-0">
-                                <strong>Vlada Boiko</strong></p>
-                            <p class="quote-title">UI/UX Designer</p>
+                        <div class="carousel-item">
+                            <p class="testims info text-sm-10 text-proxima text-18" id="secTest">My greatest asset is my ability to enthusiastically take on any challenge to produce impactful results.I champion being the voice of the consumer, imagining every possible angle of an approach to a product, gathering customer feedback and pain points, and executing on ideas from conception to QA.</p>
+                            <div class="d-flex align-items-end flex-column">
+                                <p class="quote-name mb-0">
+                                    <strong>Vlada Boiko</strong></p>
+                                <p class="quote-title">UI/UX Designer</p>
+                            </div>
                         </div>
                     </div>
-                </div>
                              <!-- Controls -->  
-                <a class="carousel-control-prev" href="#text-carousel" role="button" data-slide="prev">
-                    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
-                    <span class="sr-only">Previous</span>
-                </a>
-                <a class="carousel-control-next" href="#text-carousel" role="button" data-slide="next">
-                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
-                    <span class="sr-only">Next</span>
-                </a>
+                    <a class="carousel-control-prev" href="#text-carousel" role="button" data-slide="prev">
+                        <span class="carousel-control-prev-icon" aria-hidden="true">
+                            <img class="img-fluid" src="images/arrow-left.svg" alt="Arrow Left Icon" />
+                        </span>
+                        <span class="sr-only">Previous</span>
+                    </a>
+                    <a class="carousel-control-next" href="#text-carousel" role="button"            data-slide="next">
+                        <span class="carousel-control-next-icon" aria-hidden="true">
+                            <img class="img-fluid" src="images/arrow-left.svg" alt="Arrow Right Icon" />
+                        </span>
+                        <span class="sr-only">Next</span>
+                    </a>
+                </div>
             </div>
         </div>
     </section>
+
+    <section id="resources" class="pt-5 pb-5">
```

Comment:
@bmushtuk This section shouldn't be on this branch. Instead, you should have made the following steps:
1) Merge `test-third-section` branch into `master`.
2) Create a new branch off of `master`.
3) Create a Resources section and others
4) Try to make smaller changes per commit for easier review. Like a commit per section.

Now you need to google how to move this commit to a new branch.